scripts/lochness_create_template.py

```diff
@@ -65,6 +68,11 @@ def create_lochness_template(args):
             })
         df.to_csv(args.pii_csv)
 
+    # create pii table
```

Comment:
Are we sure this is a `pii` table? 🦅

Reply:
Nick and Holly are working on it to update the table. This is a template for the users to have a look and start editing to / from.  Just FYI, from the previous DPACC-PRECIENT & DPACC-PRONET meeting, we are decided to use this table yet.

Reply:
Sorry, here is my confusion--we are storing `lochness_sync_history` in this table but giving it a comment `create pii table` ⭕

Reply:
My bad. I've updated the comment.